realsense_camera/cfg/zr300_params.cfg

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+PACKAGE="realsense_camera"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+#             Name                                         Type      Level  Description                                   Default    Min     Max
+gen.add("enable_depth",                                    bool_t,   0,     "Enable Depth",                               True)
+gen.add("color_backlight_compensation",                    int_t,    0,     "Backlight Compensation",                     0,         0,      4)
+gen.add("color_brightness",                                int_t,    0,     "Brightness",                                 0,         0,      255)
+gen.add("color_contrast",                                  int_t,    0,     "Contrast",                                   50,        16,     64)
+gen.add("color_exposure",                                  int_t,    0,     "Exposure",                                   166,       50,     666)
+gen.add("color_gain",                                      int_t,    0,     "Gain",                                       64,        0,      256)
+gen.add("color_gamma",                                     int_t,    0,     "Gamma",                                      300,       100,    280)
+gen.add("color_hue",                                       int_t,    0,     "Hue",                                        0,         -2200,  2200)
+gen.add("color_saturation",                                int_t,    0,     "Saturation",                                 64,        0,      255)
+gen.add("color_sharpness",                                 int_t,    0,     "Sharpness",                                  50,        0,      7)
+
+# Must be set only if color_enable_auto_white_balance is disabled
+gen.add("color_white_balance",                             int_t,    0,     "White Balance",                              4600,      2000,   8000)
+
+gen.add("color_enable_auto_exposure",                      int_t,    0,     "Enable Auto Exposure",                       1,         0,      1)
+gen.add("color_enable_auto_white_balance",                 int_t,    0,     "Enable Auto White Balance",                  1,         0,      1)
+gen.add("r200_lr_auto_exposure_enabled",                   int_t,    0,     "LR Auto Exposure Enabled",                   0,         0,      1)
+gen.add("r200_lr_gain",                                    int_t,    0,     "LR Gain",                                    400,       100,    6399)
```

Comment:
@kevincwells As discussed, even if we remove the term "r200" from the dynamic config, it still appears when we read them directly using the get_settings service. Instead, will raise an issue with librealsense to rename these camera options and then make the change in the dynamic config.